Makefile

```diff
@@ -1,4 +1,6 @@
+.PHONY: install
+
 install:
 	./scripts/github-install.sh
 
-include scripts/Makefile
+include ./scripts/Makefile
```

Comment:
Is this required for Mac or Ubuntu? If the former, we'd have to change it everywhere in ```Bricks```.